dev/SplitButton/SplitButton_rs1.xaml

```diff
@@ -304,7 +303,7 @@
                                     FontSize="12"
                                     Text="&#xE70D;"
                                     VerticalAlignment="Center"
-                                    HorizontalAlignment="Right"
+                                    HorizontalAlignment="Center"
```

Comment:
Why changing the alignment?

Reply:
If I removed the right padding and instead centered it horizontally, if I leave it the way it is, it's not centered 
![image](https://user-images.githubusercontent.com/11960777/62983526-7be04800-bde4-11e9-9659-b12398d151c9.png).
The other approach is to adjust the padding, but this way is more dynamic.

Reply:
There is a design change coming to the Split Button #986 with a 1px gap between the Chevron and the Button, so having the text Centred will make sense for that.

![image](https://user-images.githubusercontent.com/22958680/61009776-b445bf80-a328-11e9-98ed-43d7a366605b.png)